sky/clouds/os_image/gcp-img.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Set the image name
+IMAGE_NAME="skypilot-gcp-gpu-debian-11-$(date +%Y%m%d)"
+
+# Launch VM
+sky launch ${IMAGE_NAME}.yaml
+
+# Create the custom image
+gcloud compute images create ${IMAGE_NAME} \
+    --source-disk=<DISK_NAME> \
+    --source-disk-zone=<VM_ZONE>
```

Comment:
Is it possible to get this information based on the cluster name we used for launching the VM, so we can make the creation image creation truely automatic?
```suggestion
    --source-disk=<DISK_NAME> \
    --source-disk-zone=<VM_ZONE>
```

Reply:
There are no APIs that output just DISK_NAME, there's only the describe API that output a bunch of info and you have to look for the exact values, so I think it's actually easily to just copy paste from cloud console

Reply:
IIRC, there can be an argument to the command that tailor the output to a specific value. Maybe worth it to check `--format="value(disks.deviceName)"`. ; )
 Some reference: https://chatgpt.com/share/66ff3a34-7210-8011-a1d9-a74a7f832d3e